reference pgroonga_condition: translate of "Syntax" section (#365)
pgroonga/pgroonga.github.io: reference/functions/pgroonga-condition.md

`index_name` is index which PGroonga refer when PGroonga execute sequential search. It's `text` type.

`column_name` is column into index which PGroonga refer when PGroonga execute sequential search. It's `text` type.

You can omission all arguments of `pgroonga_condition()`.If you want to specify specific argument, you can use [Named Notation][sql-syntax-calling-funcs-named] such as `name => value` without dependence on a position of `pgroonga_condition()` 's argument. For example, if you only specify `index_name` to argument, you can write `pgroonga_condition(index_name => 'index1')`.

However, you just remember the following three cases is enough.

```
pgroonga_condition('keyword', index_name => 'pgroonga_index')
pgroonga_condition('keyword', ARRAY[weight1, weight2, ...])
pgroonga_condition('keyword', ARRAY[weight1, weight2, ...], index_name => 'pgroonga_index')
```

Please refer [Calling Functions][sql-syntax-calling-funcs] about difference in you need to write `name => value` or not.

## Usage

## See also

* [postgres_fdw][postgres-fdw]
* [normalizers_mapping][normalizers-mapping]

> * [Calling Functions][sql-syntax-calling-funcs]
> * [Named Notation][sql-syntax-calling-funcs-named]

Reviewer: ```suggestion
* [Calling Functions][sql-syntax-calling-funcs]

* [Named Notation][sql-syntax-calling-funcs-named]
```

余計な改行っぽい感じですが、リスト表示の一個一個に改行を入れると `.po` ファイルで `msgid` が分かれるので、翻訳を共有しやすくなったりして便利らしいです。

Author: おぉ。たしかに。やってみます。

Author: リストに改行をいれました。

                                      column_name)
```

`keyword` is a keyword for full text search. It's `text` type.

> `weights` is importance factors of each value. It's `int[]` type.

Reviewer 2: ```suggestion
`weights` is importance factors for the columns being searched. It's `int[]` type.
```

Author: あ、ここも、他の場所と表現を合わせているところでした。
適用してしまいましたが、もとに戻します。

> `scorers` is [score compute procedures][scorer] of each value. It's `text[]` type.

Reviewer 2: ```suggestion
`scorers` is [scorer procedures][scorer] that compute the scores for the columns being searched. It's `text[]` type.
```

Author: "score compute procedures"は別の場所と表現を合わせているので、そのままとさせてください。

> `schema_name` is scheam which index that PGroonga refer when PGroonga execute sequential search belong to. It's `text` type.

Reviewer 2: ```suggestion
`schema_name` is the schema to which the index that PGroonga refers to when executing a sequential search belongs. It's `text` type.
```

> `index_name` is index which PGroonga refer when PGroonga execute sequential search. It's `text` type.

Reviewer 2: ```suggestion
`index_name` is index which PGroonga refer to when executing sequential search. It's `text` type.
```

> `column_name` is column into index which PGroonga refer when PGroonga execute sequential search. It's `text` type.

Reviewer 2: ```suggestion
`column_name` is the column within the index which PGroonga refers to when executing a sequential search. It's `text` type.
```

> You can omission all arguments of `pgroonga_condition()`.If you want to specify specific argument, you can use [Named Notation][sql-syntax-calling-funcs-named] such as `name => value` without dependence on a position of `pgroonga_condition()` 's argument. For example, if you only specify `index_name` to argument, you can write `pgroonga_condition(index_name => 'index1')`.

Reviewer 2: [suggestion]
```suggestion
All arguments of `pgroonga_condition()` are optional. If you want to specify a particular argument, you can use [Named Notation][sql-syntax-calling-funcs-named] such as `name => value` without relying on its position. For example, if you specify only `index_name` argument, you can write `pgroonga_condition(index_name => 'index1')`.
```

> However, you just remember the following three cases is enough.

Reviewer 2: [suggestion]
```suggestion
In general, it is enough to remember the following three cases.
```

```
pgroonga_condition('keyword', index_name => 'pgroonga_index')
pgroonga_condition('keyword', ARRAY[weight1, weight2, ...])
pgroonga_condition('keyword', ARRAY[weight1, weight2, ...], index_name => 'pgroonga_index')
```

> Please refer [Calling Functions][sql-syntax-calling-funcs] about difference in you need to write `name => value` or not.

Reviewer 2: `refer to` にするとよく利用する言い回しになると思います。
`difference in` のあとは名詞を取る印象があるので、文を持ってきて比較のときは前置詞を変えると良いかもです。
```suggestion
Please refer to [Calling Functions][sql-syntax-calling-funcs] for information about the difference between when you need to write `name => value` and when you don't.
```
